ngx-fudis/projects/ngx-fudis/src/lib/foundations/shadows/tokens.scss

```diff
@@ -1,12 +1,12 @@
 @use 'sass:map';
 @use '../colors/tokens.scss' as colors;
 
-$sisu-shadow-blue: colors.$sisu-primary 0 0 0 1px;
+$shadow-primary: var(--color-primary) 0 0 0 1px;
```

Comment:
Pitäiskö tää nimetä uudelleen? esim. fudis-shadow. Tuntuu ristiriitaiselta et nyt esim button secondarylla on primary shadow.

Reply:
Joo hyvä pointti, ei voi olla primary nimellä. Muutan tähän hetkeen nyt `fudis-shadow` nimiseksi ja katsotaan joudutaanko vielä viilaamaan näitä nimeämisiä myöhemmin.

Reply:
Eeron pullassa nämä shadows tokenit ja mixinit oli kokonaan poistettu, joten byebye.